livestatus.go

```diff
@@ -19,3 +22,10 @@ func NewLivestatus(network, address string) *Livestatus {
 		address: address,
 	}
 }
+
+// NewTLSLivestatus creates a new binding instance.
```

Comment:
The comment should start with `NewLivestatusWithDialer` and mention that it uses an external dialer.